pallets/identity/src/lib.rs

```diff
@@ -1467,7 +1467,8 @@ impl<T: Trait> Module<T> {
     ) -> Option<IdentityClaim> {
         let pk = Claim1stKey { target, claim_type };
         let sk = Claim2ndKey { issuer, scope };
-        Claims::contains_key(&pk, &sk).then(|| Claims::get(&pk, &sk))
+        let claim = Claims::get(&pk, &sk);
+        Claims::contains_key(&pk, &sk).then(|| claim)
```

Comment:
This should regress performance, so why this diff?

Reply:
(Also, this is more of a nit, relatively speaking, but from a readability POV, introducing a temporary here makes things harder to read, if ever so slightly in this location...)

Reply:
This will not be going into the production code. This is me trying to debug something.